openapi/components/schemas/WorkflowEngineVersion.yaml

```diff
@@ -0,0 +1,9 @@
+title: WorkflowEngineVersion
+type: object
+properties:
+  workflow_engine_version:
+    type: array
+    items:
+      type: string
+    description: an array of one or more acceptable engines for the `workflow_engine`
```

Comment:
For consistency, start with `A` rather than `a` and conclude with `.`

Reply:
Fixed